hackathon_site/dashboard/frontend/src/components/cart/CartCard/CartCard.test.tsx

```diff
@@ -64,4 +65,42 @@ describe("<CartCard />", () => {
             expect(getByText(error)).toBeInTheDocument();
         });
     });
+
+    test("removeFromCart button", async () => {
+        const store = makeStoreWithEntities({ hardware: mockHardware });
+
+        store.dispatch(
```

Comment:
It's good you are using the reducer you made! But it's not needed for this test since you're trying to test the remove from cart functionality. What you can do instead is populate the store with initial cart items, you'll have to modify the makeStoreWithEntities function (in utils.tsx line 100) to accept cart items.

This is not required for this ticket, but it might help having a pre-populated store for future tickets, reach out if you want help doing this!